README.md

```diff
@@ -66,14 +66,14 @@ After building Valkey, it is a good idea to test it using:
 
 The above runs the main integration tests. Additional tests are started using:
 
-    % make test-unit     # Unit tests
+    % make test-unit     # Unit tests (both C unit tests and google unit tests)
```

Comment:
Instead of "google unit tests" I'd say "gtest unit tests".  I don't think we want to emphasize Google here.  As written, it might make people think that Google wrote some of the tests.